deltachat-ios/View/Cell/DocumentGalleryFileCell.swift

```diff
@@ -5,6 +5,14 @@ class DocumentGalleryFileCell: UITableViewCell {
 
     static let reuseIdentifier = "document_gallery_file_cell"
 
+    static var cellHeight: CGFloat {
+        let textHeight = UIFont.preferredFont(forTextStyle: .headline).pointSize + UIFont.preferredFont(forTextStyle: .subheadline).pointSize + 24
+        if textHeight > 74.5 {
+            return textHeight
+        }
+        return 74.5
```

Comment:
why raise the minimum from 60 to 74.5 here?

for normal and small font sizes, this wastes much screen space so that the user does not really benefit from smaller sizes.

so, in normal font size quite fewer content is displayed: before (11.5 rows) vs after (9 rows) :
<img  width=220 src=https://user-images.githubusercontent.com/9800740/172547797-5f84fd5f-ffd5-4367-a283-f1af61f3163f.PNG> <img width=220  src=https://user-images.githubusercontent.com/9800740/172547778-6d8ed6b8-d6f2-41e9-a43a-18b35b3679f7.PNG>

not that i am not suggesting to change the size of the icon, that seems fine in all cases.

Reply:
> It's just the same cell height we're having in contact cells and chat list as well, we can reduce the height though.

i mainly came over that, as we are doing some calculations on the row height, but in practice, it is just always the same :)

i think, we should use roughly the old number of row as a reference, and go to slightly smaller/larger heights when font sizes change - i assume, this was also the gist, but at least for iphone13 mini, i end up always at the same row height.

Reply:
using a simulator you can easily switch between font sizes with command option + or -

The last changes:

![Simulator Screen Recording - iPhone 13 - 2022-06-13 at 13 29 26](https://user-images.githubusercontent.com/2614900/173344347-cd26d2fb-b503-46be-8caa-e751c3962f60.gif)